cisc/commands.go

```diff
@@ -46,6 +46,12 @@ func init() {
 				Usage:   "check the health of the cothority",
 				Action:  idCheck,
 			},
+			{
+				Name:    "qrcode",
+				Aliases: []string{"qr"},
+				Usage:   "print out the qrcode of the identity",
```

Comment:
the identity ? isn't it the configuration ? Or do I misunderstand ?
We should start to have a glossary maybe regarding the different skipchain's usage: "server identity", "configuration", etc